includes/class-sensei-lesson.php

```diff
@@ -287,8 +290,21 @@ public function lesson_prerequisite_meta_box_content() {
 			'suppress_filters' => 0,
 			'post_status'      => [ 'publish', 'draft' ],
 		);
+
+		if ( $current_lesson_course ) {
+			// Add meta query to only get Lesson Posts from current Course.
+			// phpcs:ignore WordPress.DB.SlowDBQuery
```

Comment:
Probably best to ignore the actual rule here.

```suggestion
			// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query -- Slow query ok.
```